mmv1/products/securesourcemanager/Instance.yaml

```
@@ -20,7 +20,8 @@ immutable: true
 description: 'Instances are deployed to an available Google Cloud region and are accessible via their web interface.'
 references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
-    'Official Documentation': 'https://cloud.google.com/secure-source-manager/overview/overview'
+    'Official Documentation': 'https://cloud.google.com/secure-source-manager/'
```

```suggestion
    'Official Documentation': 'https://cloud.google.com/secure-source-manager/docs/create-instance'
```

I think it would be good to point to this specific page